### PR TITLE
cmd: add Hindi Translation

### PR DESCRIPTION
**Pull request by ec-026**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.hi/windows/cmd.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`cmd`
+
+- एक आदेश निष्पादित करें:
+
+`cmd /c "{{command}}"`
+
+- एक स्क्रिप्ट निष्पादित करें:
+
+`cmd {{path/to/file.bat}}`
```
Comment (kbdharun) [inline pages.hi/windows/cmd.md:16]:
```suggestion
`cmd {{फ़ाइल.bat/का/पथ}`
```

File: pages.hi/windows/cmd.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- एक कमांड निष्पादित करें और फिर एक इंटरेक्टिव शेल दर्ज करें:
+
+`cmd /k "{{command}}"`
```
Comment (kbdharun) [inline pages.hi/windows/cmd.md:20]: Translate `{{command}}` too

File: pages.hi/windows/cmd.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+> विंडोज कमांड दुभाषिया।
+> अधिक जानकारी: <https://learn.microsoft.com/windows-server/administration/windows-commands/cmd>।
+
+- एक इंटरैक्टिव शेल सत्र प्रारंभ करें:
+
+`cmd`
+
+- एक आदेश निष्पादित करें:
+
```
Comment (kbdharun) [inline pages.hi/windows/cmd.md:11]: Same here translate `{{command}}`

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, I have some suggestions for this page.

File: pages.hi/windows/cmd.md (hunk 3 above)
Comment (ec-026) [reply pages.hi/windows/cmd.md:11]: All the changes have been made as requested. Thanks for the review😊.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**